modules/administration-guide/partials/proc_configuring-che-with-custom-resources-to-use-amazon-s3.adoc

```diff
@@ -0,0 +1,62 @@
+[id="configuring-{prod-id-short}-with-custom-resources-to-use-amazon-s3_{context}"]
+= Configuring {prod-short} with custom resources to use Amazon S3
+
+To configure {prod-short} to use Amazon S3:
+
+.Prerequisites
+
+* Configured Amazon S3. See xref:supported-restic-compatible-backup-servers.adoc[].
+
+.Procedure
+
+. Create a secret containing the `repo-password` key with a Base64 encoded password:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backup-encryption-password-secret
+  namespace: eclipse-che
+type: Opaque
+stringData:
+  repo-password: __<base64_encoded_password>__
```

Comment:
This brings unneeded complexity, please fix. One may use `data` field with direct (not base64 encoded) values ([docs](https://kubernetes.io/docs/concepts/configuration/secret/#overview-of-secrets)).
The same for all other backup procedures.

Reply:
Alright. Fixed.